ccdaservice/ccda_gateway.php

```diff
@@ -27,6 +27,7 @@
 } else {
     OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
     $ignoreAuth = false;
+    $sessionAllowWrite = true;
```

Comment:
TODO, need to look into this more. if session has already been set up previously for cor openemr use, then no need for this.